ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -265,6 +267,8 @@ public sealed class Commands {
 							return await ResponsePlay(steamID, args[1], Utilities.GetArgsAsText(message, 2)).ConfigureAwait(false);
 						case "PLAY":
 							return await ResponsePlay(steamID, args[1]).ConfigureAwait(false);
+						case "POINTS":
+							return await ResponsePointsBalance(steamID, args[1]).ConfigureAwait(false);
```

Comment:
Consider using `Utilities.GetArgsAsText` because the last argument is the `botNames`.